# CVE-2022-34265

An issue was discovered in Django 3.2 before 3.2.14 and 4.0 before 4.0.6. The Trunc() and Extract() database functions are subject to SQL injection if untrusted data is used as a kind/lookup_name value. Applications that constrain the lookup name and kind choice to a known safe list are unaffected.

{% embed url="https://github.com/Live-Hack-CVE/full_database/blob/main/cves/2022/CVE-2022-34265.json" %}


* coco0x0a ~> [CTF_Django_CVE-2022-34265](https://www.alice-snow.ru/2022/database/cve-2022-34265/ctf_django_cve-2022-34265-coco0x0a)
* aeyesec ~> [CVE-2022-34265](https://www.alice-snow.ru/2022/database/cve-2022-34265/cve-2022-34265-aeyesec)
* ZhaoQi99 ~> [CVE-2022-34265](https://www.alice-snow.ru/2022/database/cve-2022-34265/cve-2022-34265-zhaoqi99)
* traumatising ~> [CVE-2022-34265](https://www.alice-snow.ru/2022/database/cve-2022-34265/cve-2022-34265-traumatising)
* coco0x0a ~> [CTF_Django_CVE-2022-34265](https://www.alice-snow.ru/2022/database/cve-2022-34265/ctf_django_cve-2022-34265-coco0x0a)
* aeyesec ~> [CVE-2022-34265](https://www.alice-snow.ru/2022/database/cve-2022-34265/cve-2022-34265-aeyesec)
* ZhaoQi99 ~> [CVE-2022-34265](https://www.alice-snow.ru/2022/database/cve-2022-34265/cve-2022-34265-zhaoqi99)
* traumatising ~> [CVE-2022-34265](https://www.alice-snow.ru/2022/database/cve-2022-34265/cve-2022-34265-traumatising)
* coco0x0a ~> [CTF_Django_CVE-2022-34265](https://www.alice-snow.ru/2022/database/cve-2022-34265/ctf_django_cve-2022-34265-coco0x0a)